addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -705,3 +705,11 @@ a.no-decoration {
 .o_shadow_animation {
     box-shadow: 0 0 5px 10px rgba(240,8,0,0.4)!important;
 }
+
+.o_search_product_item {
+    min-width: 300px;
+}
```

Comment:
Is your task working in mobile ?

Reply:
It works on mobile, at least when I tested on a 360x640